src/hotspot/share/oops/method.hpp

```diff
@@ -815,6 +815,14 @@ class Method : public Metadata {
     return _method_counters;
   }
 
+  // Clear the flags related to compiler directives that were set by the compilerBroker,
+  // because the directives can be updated.
+  void clear_method_flags() {
```

Comment:
The function name is a bit misleading - it clears only flags related to directives.

Reply:
Changed to `clear_directive_flags`.